### PR TITLE
DSD-4144: Add variable for PG version

### PR DESCRIPTION
**Pull request by zbpvarun-tamr**

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -44,6 +44,7 @@ module "cloud_sql" {
 
   enable_private_ip  = var.enable_cloud_sql_private_ip
   private_network_id = var.private_network_id
+  postgres_verion    = var.sql_pg_version
```
Comment (karankaran1908) [inline main.tf:47]: @zbpvarun-tamr we have to add `postgres_version` in the `terraform-gcp-tamr-cloud-sql` module as well.

Comment (karankaran1908) [reply main.tf:47]: @zbpvarun-tamr you have to upgrade the module version to `v6.1.0`(The newer version)

Comment (zbpvarun-tamr) [reply main.tf:47]: Thanks @karankaran1908. Just pushed another commit.

**Review by karankaran1908 — APPROVED** [review]

**Pull request merged**